### PR TITLE
docs: initial cut of devdocs

### PR DESCRIPTION
**Pull request by glasnt**

### CODE REVIEW

**Review by apeabody — APPROVED** [review]

File: app/README.md
Comment (NimJay) [inline app/README.md:20]: Should this be:
```suggestion
Each language includes a `makefile` file that includes common development steps:
```

**Review by NimJay — APPROVED** [review]
Approved. Well written.

Feel free to ignore my comments.

**Review by NimJay — APPROVED** [review]
In case you're waiting on some approval: LGTM. :)

**Pull request merged**
